libraries/SuplaDevice/supla-common/proto.h

```diff
@@ -92,14 +92,15 @@ extern char sproto_tag[SUPLA_TAG_SIZE];
 
```

Comment:
Do not change supla-common files from this level. Change first
https://github.com/SUPLA/supla-core/tree/develop/supla-common, and then copy them to the arduino repository.